recipes/hpp-fcl/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir, collect_libs
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class HppFclConan(ConanFile):
+    name = "hpp-fcl"
+    description = "An extension of the Flexible Collision Library"
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/humanoid-path-planner/hpp-fcl"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_py_interface": [True, False],
+        "install_documentation": [True, False],
+        "turn_assert_into_exception": [True, False],
+        "enable_logging": [True, False],
+        "has_qhull": [True, False],
+        "has_octomap": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_py_interface": False,
+        "install_documentation": False,
+        "turn_assert_into_exception": False,
+        "enable_logging": False,
+        "has_qhull": False,
+        "has_octomap": False
+    }
+
+    def requirements(self):
+        self.requires("eigen/[>=3.4.0]", package_id_mode="minor_mode")
+        self.requires("boost/1.76.0")
+        self.requires("assimp/5.4.1")
+        if self.options.has_octomap:
+            self.requires("octomap/1.10.0")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_PYTHON_INTERFACE"] = self.options.with_py_interface
```

Comment:
Well, I've tried to pretend this dependency doesn't exist since the package is intended for C/C++ devs 🤔 
But in the recipe implementation, this option is supported and set to `False`, unlike the original **hpp-fcl/CMakeLists.txt**.
So the package building would succeed, unless `... -o with_py_interface=True` is provided.

Would you like me to remove the support for the `BUILD_PYTHON_INTERFACE` option from the recipe by hardcoding it to `False`?

Reply:
Maybe you could just use `tc.variables["BUILD_PYTHON_INTERFACE"] = False`, then ?

Reply:
Done ✅